contracts/interfaces/IAmpleforth.sol

```diff
@@ -0,0 +1,25 @@
+pragma solidity 0.4.24;
+
+// Public interface definition for the Ampleforth supply policy on Ethereum (the base-chain)
+interface IAmpleforth {
+    function rebase() external;
+
+    function epoch() external view returns (uint256);
+
+    function lastRebaseTimestampSec() external view returns (uint256);
+
+    function inRebaseWindow() external view returns (bool);
+
+    event LogRebase(
```

Comment:
Does it make sense to include events in interaces? Only offchain services have access to logs, and they'd use an abi instead of a solidity interface.

Reply:
Makes sense 👍